itl/command-plugins-windows.conf

```diff
@@ -317,3 +317,31 @@ object CheckCommand "users-windows" {
 		}
 	}
 }
+
+object CheckCommand "file-age-windows" {
+        command = [ PluginDir + "/check_file_age.cmd" ]
+
+        arguments = {
+                "file" = {
+                        skip_key = true
+                        order = 0
+                        value = "$file_age_file$"
```

Comment:
This might interfere the `file_age` check command for the Linux. Please rename this to `$file_age_win_file$`.

Reply:
Okay I will do that!